plugin/trino-druid/src/test/java/io/trino/plugin/druid/BaseDruidConnectorTest.java

```diff
@@ -70,6 +70,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_CREATE_TABLE:
             case SUPPORTS_CREATE_TABLE_WITH_DATA:
             case SUPPORTS_ADD_COLUMN:
+            case SUPPORTS_DROP_COLUMN:
```

Comment:
ADD and DROP column should be supported together.
in rare cases, eg Delta, DROP is not going to be supported even wehn ADD is supported, so we need two testing behaviors, but in common case, we should have it simpler

can you please cherry https://github.com/trinodb/trino/pull/13897 here to avoid these changes?